src/commcare_cloud/ansible/roles/backups/templates/check_s3_backup.py.j2

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
```

Comment:
This could use a comment explaining how it works. It's not clear why it's showing an alert if the key is older than 2 days. I'm assuming the same key get's overwritten each time so an old key indicates an out of date backup?

Reply:
it checks the last modified date, ALERT means that backup has not been uploaded for 2 days.